### ISSUE TITLE
error installing mrp_mrp

### ISSUE THREAD

**Issue by mauroanelli2017** [asphaltzipper/azi-odoo-modules#13]
Dear Mr,
while istalling on v10 ce
mrp_master_schedule
mrp_llc
mrp_mrp
simulated_pick
i got this error
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/odoo/http.py", line 640, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/usr/lib/python2.7/dist-packages/odoo/http.py", line 677, in dispatch
    result = self._call_function(**self.params)
  File "/usr/lib/python2.7/dist-packages/odoo/http.py", line 333, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/odoo/service/model.py", line 101, in wrapper
    return f(dbname, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/odoo/http.py", line 326, in checked_call
    result = self.endpoint(*a, **kw)
  File "/usr/lib/python2.7/dist-packages/odoo/http.py", line 935, in __call__
    return self.method(*args, **kw)
  File "/usr/lib/python2.7/dist-packages/odoo/http.py", line 506, in response_wrap
    response = f(*args, **kw)
  File "/usr/lib/python2.7/dist-packages/odoo/addons/web/controllers/main.py", line 889, in call_button
    action = self._call_kw(model, method, args, {})
  File "/usr/lib/python2.7/dist-packages/odoo/addons/web/controllers/main.py", line 877, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/usr/lib/python2.7/dist-packages/odoo/api.py", line 689, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/usr/lib/python2.7/dist-packages/odoo/api.py", line 680, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/odoo/addons/base/module/module.py", line 411, in button_immediate_install
    return self._button_immediate_function(type(self).button_install)
  File "/usr/lib/python2.7/dist-packages/odoo/addons/base/module/module.py", line 485, in _button_immediate_function
    modules.registry.Registry.new(self._cr.dbname, update_module=True)
  File "/usr/lib/python2.7/dist-packages/odoo/modules/registry.py", line 82, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/usr/lib/python2.7/dist-packages/odoo/modules/loading.py", line 339, in load_modules
    loaded_modules, update_module)
  File "/usr/lib/python2.7/dist-packages/odoo/modules/loading.py", line 237, in load_marked_modules
    loaded, processed = load_module_graph(cr, graph, progressdict, report=report, skip_modules=loaded_modules, perform_checks=perform_checks)
  File "/usr/lib/python2.7/dist-packages/odoo/modules/loading.py", line 156, in load_module_graph
    _load_data(cr, module_name, idref, mode, kind='data')
  File "/usr/lib/python2.7/dist-packages/odoo/modules/loading.py", line 95, in _load_data
    tools.convert_file(cr, module_name, filename, idref, mode, noupdate, kind, report)
  File "/usr/lib/python2.7/dist-packages/odoo/tools/convert.py", line 845, in convert_file
    convert_xml_import(cr, module, fp, idref, mode, noupdate, report)
  File "/usr/lib/python2.7/dist-packages/odoo/tools/convert.py", line 915, in convert_xml_import
    obj.parse(doc.getroot(), mode=mode)
  File "/usr/lib/python2.7/dist-packages/odoo/tools/convert.py", line 799, in parse
    self._tags[rec.tag](rec, de, mode=mode)
  File "/usr/lib/python2.7/dist-packages/odoo/tools/convert.py", line 501, in _tag_menuitem
    menu_parent_id = self.id_get(rec.get('parent',''))
  File "/usr/lib/python2.7/dist-packages/odoo/tools/convert.py", line 782, in id_get
    res = self.model_id_get(id_str, raise_if_not_found)
  File "/usr/lib/python2.7/dist-packages/odoo/tools/convert.py", line 788, in model_id_get
    return self.env['ir.model.data'].xmlid_to_res_model_res_id(id_str, raise_if_not_found=raise_if_not_found)
  File "/usr/lib/python2.7/dist-packages/odoo/addons/base/ir/ir_model.py", line 1076, in xmlid_to_res_model_res_id
    return self.xmlid_lookup(xmlid)[1:3]
  File "<decorator-gen-5>", line 2, in xmlid_lookup
  File "/usr/lib/python2.7/dist-packages/odoo/tools/cache.py", line 87, in lookup
    value = d[key] = self.method(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/odoo/addons/base/ir/ir_model.py", line 1065, in xmlid_lookup
    raise ValueError('External ID not found in the system: %s' % xmlid)
ParseError: "External ID not found in the system: mrp_workorder.mrp_planning_menu_root" while parsing /usr/lib/python2.7/dist-packages/odoo/addons/mrp_mrp/views/mrp_material_plan_view.xml:156, near
<menuitem id="mrp_material_plan_menu" name="Material Plan" sequence="3" action="mrp_material_plan_action" parent="mrp_workorder.mrp_planning_menu_root" groups="stock.group_stock_user"/>

**Comment by matt454357** [comment]
Oh, yeah.  The mrp_workorder module is from the enterprise edition.  I will find a way to make that work with community.

**Comment by mauroanelli2017** [comment]
thank you very much

**Comment by matt454357** [comment]
As a workaround, you can remove lines 151 through 156 in the file mrp_mrp/views/mrp_material_plan_view.xml:

```xml
<menuitem id="mrp_material_plan_menu"
    name="Material Plan"
    sequence="3"
    action="mrp_material_plan_action"
    parent="mrp_workorder.mrp_planning_menu_root"
    groups="stock.group_stock_user"/>
```
We create a menu item in Inventory -> Planning for the same view.

**Comment by matt454357** [comment]
Fixed at 0e4e913
